modules/test/playwright/tests/frontend-js-spa-web/settings.spec.ts

```diff
@@ -73,3 +73,69 @@ test(
 		});
 	}
 );
+
+test(
+	'Can change user notification timeout',
+	{tag: '@LPS-67072'},
+	async ({page, systemSettingsPage}) => {
+		await test.step('Navigate to SPA Settings page', async () => {
+			await systemSettingsPage.goToSystemSetting(
+				'Infrastructure',
+				'Frontend SPA Infrastructure'
+			);
+		});
+
+		const userNotificationTimeoutLabel = page.getByLabel(
```

Comment:
We have only a few SPA settings page locator, but we can consider creating a `SettingsPage.ts` to hold all of them if the number increase.

Reply:
Yep. Very optional for this PR, can be a rule of three.